playpen/vagrant-setup.sh

```diff
@@ -47,8 +47,10 @@ debug: true
 endpoint: pulp-devel:5001
 EOF
 
-    mkdir -p metadata
-    sudo ln -s $HOME/devel/crane/metadata/ /var/lib/pulp/published/docker/app
+    mkdir -p metadata/v1 metadata/v2
+    sudo mkdir -p /var/lib/pulp/published/docker/v1 /var/lib/pulp/published/docker/v2
```

Comment:
is not it supposed to be '/var/lib/pulp/published/docker/v1/app'? same for v2 version.